uint8 MOWER_STATUS_INITIALIZING=0
uint8 MOWER_STATUS_OK=255

time stamp

uint8 mower_status
bool rain_detected
#float32[5] ultrasonic_ranges
bool emergency
float32 v_charge
float32 v_battery
float32 charge_current
float32 mower_temp
float32 esc_mower_temp
float32 esc_mower_left
float32 esc_mower_right


